### ISSUE TITLE
Phenotypes pertaining to mixed venous oxygen saturation, peripheral oxygen saturation and pCO2 gap

### ISSUE THREAD

**Issue by nklsbckmnn** [obophenotype/human-phenotype-ontology#11232]
**Preferred term label:**

1. Abnormal mixed venous oxygen saturation (SvO2) 
2. Increased mixed venous oxygen saturation (SvO2) 
3. Decreased mixed venous oxygen saturation (SvO2) 

4. Decreased peripheral oxygen saturation (SO2)

6. Increased pCO2 gap

**Synonyms**

**Definition (free text, please give PubMed ID)**

1. An abnormal blood oxygen level in the form of an abnormal oxygen saturation in mixed venous blood (SvO2) [Comment: Can be approximated by oxygen saturation in central venous blood.]
2. An abnormal mixed venous oxygen saturation that is increased (SvO2 > 80 %)
3. An abnormal mixed venous oxygen saturation that is decreased (SvO2 < 60-65 %)

4. An abnormal blood oxygen level in the form of a decreased oxygen saturation in capillary blood (SO2 < 95 %)

5. An abnormal blood carbon dioxide level in the form an increased gap between central venous or mixed venous pvCO2 and arterial paCO2 (pvCO2-paCO2 > 6 mmHg or 0.8 kPa)

**Parent term (use hpo.jax.org/app)**

1. Abnormal blood oxygen level (HP_0500165)
2. <1.>
3. <1.>

4. HP_0500165

5. Abnormal blood carbon dioxide level (HP_0500164)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@nklsbckmnn 
Adding the following, please respond if the definition needs revision. We usually try to provide a little more context with the definitions:

Adding new term: 
- ID: HP:6001379
- label: Abnormal mixed venous oxygen saturation
- parents: [Abnormal blood oxygen level (HP:0500165)](https://hpo.jax.org/app/browse/term/HP:0500165)
- pmid: PMID:6870412
- def: Abnormal oxygen saturation in mixed venous blood (SvO2). SvO2 can be measured in a sample of mixed venous blood drawn from the distal lumen of a catheter placed in the pulmonary artery. The SvO2 is dependent on arterial oxygen saturation, hemoglobin concentration, cardiac output, and tissue oxygen demands.
- comment: A fall in SvO2 usually implies anemia, arterial oxygen desaturation, and/or decreased cardiac output; however, a normal or high value does not exclude such disturbances. When employed in conjunction with the other indicators of tissue oxygenation available in an intensive care unit, SvO2 can be useful as a guide for both prognosis and urgency of therapy.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11232
- synonyms: 

I will add the remaining terms next week!

**Comment by nklsbckmnn** [comment]
Thanks so much for looking into this. Feel free to adopt or ignore...

def: Abnormal oxygen saturation in mixed venous blood (SvO2). SvO2 can be measured in a sample of mixed venous blood drawn from the distal lumen of a catheter placed in the pulmonary artery. The SvO2 is dependent on O2 delivery to tissue (DO2) and tissue O2 demand (VO2). Oxygen delivery to tissue is dependent on arterial oxygen content (CaO2) and cardiac output (CO). Arterial oxygen content is dependent on arterial oxygen saturation (hemoglobin saturation and dissolved oxygen) as well as hemoglobin concentration.

comment: A decrease of SvO2, holding other factors constant, correlates with anemia, arterial oxygen desaturation, and/or decreased cardiac output; however, a normal or high value does not exclude such disturbances. An increase of SvO2, holding other factors constant, correlates with increased O2 delivery to tissue (increased FiO2, hyperoxia, hyperbaric oxygen), decreased tissue O2 demand (hypothermia, anaesthesia, neuromuscular blockade) and/or high flow states (sepsis, hyperthyroidism, severe liver disease). When employed in conjunction with the other indicators of tissue oxygenation available in an intensive care unit, SvO2 can be useful as a guide for both prognosis and urgency of therapy. SvO2 correlates with oxygen saturation in central venous blood (ScvO2). ScvO2 (in blood from whole body including heart) is usually higher than SvO2 if sampled from the superior vena cava (blood from upper body) and usually lower than SvO2 if sampled from the inferior vena cava (blood from lower body) due to differences in oxygen demand of different body parts.

Took inspiration from https://litfl.com/mixed-venous-oxygen-saturation-svo2-monitoring/

**Comment by pnrobinson** [comment]
@nklsbckmnn 
Adding new term: 
- ID: HP:6001450
- label: Increased mixed venous oxygen saturation
- parents: [Abnormal mixed venous oxygen saturation (HP:6001379)](https://hpo.jax.org/app/browse/term/HP:6001379)
- pmid: PMID:6870412
- def: Oxygen saturation in mixed venous blood (SvO2), as measured in a sample of mixed venous blood drawn from the distal lumen of a catheter placed in the pulmonary artery, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11232
- synonyms:

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001451
- label: Decreased mixed venous oxygen saturation
- parents: [Abnormal mixed venous oxygen saturation (HP:6001379)](https://hpo.jax.org/app/browse/term/HP:6001379)
- pmid: PMID:6870412
- def: Oxygen saturation in mixed venous blood (SvO2), as measured in a sample of mixed venous blood drawn from the distal lumen of a catheter placed in the pulmonary artery, is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11232
- synonyms:

**Comment by pnrobinson** [comment]
@nklsbckmnn 
Returning to this, I am wondering how to relate Decreased peripheral oxygen saturation (SO2) to the existing term

[Hypoxemia HP:0012418](https://hpo.jax.org/browse/term/HP:0012418)

If we just say "Hypoxemia" clinically, then in the vast majority of cases, we mean "peripheral saturation", correct?

**Comment by pnrobinson** [comment]
@nklsbckmnn 
Adding new term: 
- ID: HP:6001453
- label: Elevated pCO2 gap
- parents: [Abnormal blood carbon dioxide level (HP:0500164)](https://hpo.jax.org/app/browse/term/HP:0500164)
- pmid: PMID:32377433
- def: The difference between the venous-to-arterial carbon dioxide saturation (Pv-aCO2) is above the upper limit of normal.
- comment: Central venous-arterial carbon dioxide difference (PCO2 gap) can be a marker of cardiac output adequacy in global metabolic conditions that are less affected by the impairment of oxygen extraction capacity.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11232
- synonyms: Increased pCO2 gap(NONE)

**Comment by nklsbckmnn** [comment]
I would define hypoxemia either as decreased paO2 or as disjunction between decreased paO2 and decreased SaO2 (SpO2 when measured in the periphery). I would not define it as decreased SpO2. In some cases SaO2/SpO2 and paO2 do not correlate. SaO2 is low while paO2 can be normal in e. g. carbon monoxide poisoning. paO2 can start to fall while SaO2 still remains normal. So they are not identical and they are not logical or nomological subclasses of one another. SaO2 and SpO2 can come apart in clinically meaningful ways, too.

I would suggest keeping hypoxemia as disjunction between (superclass above) decreased paO2 and decreased SpO2 (and possibly others like decreased SaO2 and decreased CaO2 that could be added (later) if needed).

* Hypoxemia
** Decreased SpO2
** Decreased paO2

or even

* Hypoxemia
** Decreased hemoglobin oxygen saturation
*** Decreased SpO2
(*** Decreased SaO2)
** Decreased paO2
(** Decreased CaO2)

Some examples of definitions of "hypoxemia" compatible with disjunction:
* The current definition of hypoxemia in HPO (but not the comment that refers to paO2 alone)
* https://www.mayoclinic.org/symptoms/hypoxemia/basics/definition/sym-20050930
* https://en.wiktionary.org/wiki/hypoxemia
* https://www.merriam-webster.com/dictionary/hypoxemia#medicalDictionary
* Stedman's Medical Dictionary ("subnormal oxygenation of arterial blood,
short of anoxia", cited in https://www.researchgate.net/publication/342191443_Why_COVID-19_Silent_Hypoxemia_is_Baffling_to_Physicians/link/5ee82284458515814a627ec8/download?_tp=eyJjb250ZXh0Ijp7ImZpcnN0UGFnZSI6InB1YmxpY2F0aW9uIiwicGFnZSI6InB1YmxpY2F0aW9uIn19)
*  Dorland's Illustrated Medical Dictionary 908 (32d ed. 2012) ("deficient oxygenation of the blood", as cited in https://www.va.gov/vetapp13/Files1/1310355.txt)
